app/assets/stylesheets/mobile/lightbox.scss

```diff
@@ -0,0 +1,19 @@
+.lightbox:hover .meta {
+  opacity: .8;
+}
+
+.meta {
+  height: 25px;
+  width: 25px;
+  bottom: 0;
+  left: 0;
+
+  .filename, .informations {
+    display: none;
+  }
+
+  .expand {
```

Comment:
I noticed you changed these values. Out of curiosity, do these new values show the expand icon centered within the grey box on the mobile devices/browsers you have checked? When I look at it on my phone it appears a little offset. Check out the screenshots below. I've got what looks centered for me on the left and then the current values in the PR on the right. If it's not consistent across different mobile browsers, it might be best to find a solution that is.

![hover](https://user-images.githubusercontent.com/22733864/44453445-ec77ce80-a5ad-11e8-86d3-861178cdd882.png)

Reply:
Yeah, centered looks better 👍

Reply:
Ha. I do not have an iPhone and tested using Chrome DevTools. Your coordinates looked a little off, similar to how the new coordinates look to you.

Since you have first-hand experience, I will use your coordinates. Thank you for your feedback, I appreciate it!